stdlib/cgi.pyi

```diff
@@ -47,15 +47,17 @@ class MiniFieldStorage:
     list: Any
     type: Any
     file: Optional[IO[bytes]]
-    type_options: Dict[Any, Any]
+    type_options: dict[Any, Any]
     disposition: Any
-    disposition_options: Dict[Any, Any]
-    headers: Dict[Any, Any]
+    disposition_options: dict[Any, Any]
+    headers: dict[Any, Any]
     name: Any
     value: Any
     def __init__(self, name: Any, value: Any) -> None: ...
     def __repr__(self) -> str: ...
 
+_list = list
```

Comment:
Is this really necessary? It seems a bit weird to me.

Reply:
Is it maybe because of this?

```python
list: Optional[_list[Any]]
```

Reply:
Yes, without the type alias, there is an error, since `list` is aliased by this.